tests/test_snapshots.py

```
@@ -1111,6 +1111,29 @@ def test_orch_snapshot_disabled_skips_entirely(tmp_path, cache_manager):
     orch.client_app_downloader.fetch_snapshot_release.assert_not_called()
 
 
+@pytest.mark.integration
+def test_orch_snapshot_disabled_reports_existing_snapshot_state(tmp_path, cache_manager):
```

Ruff (0.16.0)

[warning] Unused function argument: `cache_manager`

(ARG001)